### PR TITLE
Prepare pypi release

### PR DESCRIPTION
**Pull request by SilverBzH**

### CODE REVIEW

**Review by uael — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -11,6 +17,9 @@ dependencies = [
     "bitstruct>=8.12",
 ]
 
+[project.urls]
+"Source" = "https://github.com/google/avatar"
```
Comment (DeltaEvo) [inline pyproject.toml:21]: nit: Do we need the quotes here ?

File: pyproject.toml [hunk 2]
```diff
@@ -1,8 +1,14 @@
 [project]
 name = "avatar"
+version = "0.0.1"
 authors = [{name = "Pandora", email = "pandora-core@google.com"}]
+description = "Avatar aims to provide a scalable multi-platform Bluetooth testing tool capable of running any Bluetooth test cases virtually and physically."
 readme = "README.md"
-dynamic = ["version", "description"]
```
Comment (DeltaEvo) [inline pyproject.toml:5]: Why are you removing the dynamic version and description ?

**Review by DeltaEvo — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (DeltaEvo) [reply pyproject.toml:21]: Looking at the doc it seems that we don't https://flit.pypa.io/en/latest/pyproject_toml.html#urls-table

```suggestion
Source = "https://github.com/google/avatar"
```

Comment (SilverBzH) [reply pyproject.toml:21]: I just followed the tutorial and the quotes were here
https://packaging.python.org/en/latest/tutorials/packaging-projects/#configuring-metadata

I'll remove them, looks cleaner without

File: pyproject.toml (hunk 2 above)
Comment (DeltaEvo) [reply pyproject.toml:5]: Let me rephrase the question then, why do we don't want them to be dynamic ?

**Review by DeltaEvo — APPROVED** [review]
